source/install/troubleshooting.rst

```diff
@@ -19,9 +19,40 @@ General Troubleshooting
 -----------------------
 Some of these suggestions can be done directly, and others may need consultation from your network administrator.
 
-- Take a look at the logs (``mattermost.log`` and NGINX logs) for errors.
-- You can also search the error messages online - existing solutions can often be applied.
-- Open **System Console > General > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space.
+Take a Look at the Logs
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This article covers accessing the numerous logs created by Mattermost applications.
+
+**Mattermost Server**
+
+1. In the *System Console* go to *Environment > Logging* and note down the path to the *File Log Directory*
+2. The server log file is called *mattermost.log* and can be opened with a standard text editor or shared directly.
+
+.. Note::
+   Make sure to verify **logging to a file** is enabled by confirming **System Console > Environment > Logging > Output logs to file** is set to **true**. Open **System Console > Environment > Logging** and set File Log Level to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space. Logging can be set to output plain text or JSON here as well.
```

Comment:
```suggestion
Verify that logging is enabled by navigating to **System Console > Environment > Logging** and confirming that **Output logs to file** is set to **true**. 
```
I wonder if this should be moved to above step 1, and then this "Open **System Console > Environment > Logging** and set **File Log Level** to **DEBUG**. Make sure to revert to **INFO** after troubleshooting to save disk space. Logging can be set to output plain text or JSON here as well." is the content in the note?

Reply:
I think that makes more sense, it seems like part of the process, however, I am wondering if we need to be more explicit in stating, once you do this, then try to replicate the issue again, so that it is logged at the debug level.